x-pack/plugins/actions/server/builtin_action_types/servicenow/index.ts

```diff
@@ -151,6 +155,7 @@ export function getServiceNowITOMActionType(
       secrets: schema.object(ExternalIncidentServiceSecretConfiguration, {
         validate: curry(validate.secrets)(configurationUtilities),
       }),
+      connector: validate.connector,
```

Comment:
Added connector validation function in order to validate the contents config and secrets together since now the fields required in secrets depends on isOAuth in config